README.md

```diff
@@ -9,12 +9,17 @@ A web application for the UWA Historical Society. The app will use a map of the
 Find the meeting agendas and minutes at the [wiki](https://github.com/LightXEthan/uwahs-campus-map/wiki).
 
 ## Scripts
-
 - `npm install` to install dependencies. Required to do before anything else.
 - `npm start` to run in development mode. Open [http://localhost:3000](http:localhost:3000) to view it in the browser.
 - `npm test` to launch the test runner in interactive watch mode.
 - `npm run build` to build the app for production. Find it in the `/build` folder.
 
+## Getting started
+1. Install [NodeJS](https://nodejs.org/)
+2. `git clone https://github.com/LightXEthan/uwahs-campus-map` to download the files
+3. `npm install` to install the dependencies, node_modules file will appear with the dependencies
+4. `npm start` to start a local server
```

Comment:
there needs to be an `.env` file with all environment variables (api keys, database uri etc.) before starting the server, otherwise there'll be errors.